packages/core/echo/echo-db/src/echo-handler/clone.ts

```diff
@@ -32,16 +32,16 @@ const requireAutomergeCore = (obj: EchoReactiveObject<any>) => {
  * Returns new unbound clone of the object.
  * @deprecated
  */
-export const clone = <T extends {}>(
-  obj: EchoReactiveObject<T>,
+export const clone = <T extends BaseObject>(
```

Comment:
this is a good increment, but I think we should start typing those functions as:

```ts
export const clone = <O extends BaseObject>(obj: O): O
```

Reply:
just changing T to O?